docs/docsite/rst/guide_packet.rst

```diff
@@ -185,7 +185,7 @@ As with most Ansible modules, the default states of the Packet modules are idemp
 
 The second module call provisions 3 Packet Type 0 (specified using the 'plan' parameter) servers in the project identified via the 'project_id' parameter. The servers are all provisioned with CoresOS beta (the 'operating_system' parameter) and are customized with cloud-config user data passed to the 'user_data' parameter.
 
-The ``packet_device`` module has a boolean 'wait' parameter that defaults to 'false'. If set to 'true', Ansible will wait until the GET API call for a device will contain an Internet-routeable IP address. The 'wait' parameter allows us to use the IP address of the device as soon as it's available.
+The packet_device module has a ``wait_for_public_IPv`` parameter. If set to 4 or 6, Ansible will wait until the GET API call for a device will contain an Internet-routeable IP address of the respective version. If we want to refer to an IP address in consequent module calls, it's wise to use the ``wait_for_public_IPv`` parameter.
```

Comment:
Thanks @t0mk! A few review comments:

- Be explicit about what the parameter is. Suggest revising the first sentence to something like "The ``wait_for_public_IPv`` parameter is used to specify the IP version of the IP address to wait for. This value can be 4 or 6."
- s/"will contain"/"contains"
- "the respective version" is a bit hard to parse. Please rephrase.
- Please revise "If we want to..." to something like "To refer to an IP address is subsequent module calls..."

Reply:
fixed